src/test/java/com/karankumar/bookproject/ui/statistics/StatisticsViewTest.java

```diff
@@ -77,6 +83,14 @@ void createCompleteStatisticsView() {
         thereAreNotOtherStatistics();
     }
 
+    @Test
+    void shouldShowErrorMessageWhenNullpointerException() {
+        Assertions.assertThrows(NullPointerException.class, () -> {
+            UI.getCurrent().navigate(StatisticsView.class);
+          });
+    }
```

Comment:
Instead of testing the message shown when we get a null pointer exception, could you please test that no statistics are shown when there are no read books?

Reply:
I will change the test scenario to check no statistic instead of checking message